compliance_suite/report_runner.py

```diff
@@ -195,13 +204,52 @@ def test_drs_object_info(
 
     add_common_test_cases(
         test_object = drs_object_test,
-        endpoint_name = "DRS Object Info",
+        endpoint_name = endpoint_name,
         response = response,
         expected_status_code = expected_status_code,
         expected_content_type = expected_content_type,
         schema_dir = schema_dir,
         schema_file = schema_file)
 
+    # Response with expand parameter set to true
+    response = send_request(
+        server_base_url,
+        DRS_OBJECT_INFO_URL + drs_object_id,
+        headers,
+        auth_type,
+        drs_object_passport_map = drs_object_passport_map,
+        drs_object_id = drs_object_id,
+        params = {'expand': True})
+
+    # TODO: find way to determine if object is bundle or to be skipped
+    # Skip test if contents field is not available
+    if is_bundle:
+        ### CASE: response expand bundle
+        add_test_case_common(
+            test_object = drs_object_test,
+            case_type = "expand_bundle",
+            case_name = "DRS Access expand bundle validation",
+            case_description = f"Validate DRS bundle when expand = True",
+            response = response,
+            schema_name = os.path.join(schema_dir, 'drs_bundle.json'))
```

Comment:
Add a constant `DRS_BUNDLE_SCHEMA = "drs_bundle.json"` to compliance_suite/constants.py and use the constant here